code/modules/chemistry/Reagents-FoodDrink.dm

```diff
@@ -1821,6 +1834,17 @@ datum
 			reagent_state = LIQUID
 			taste = "overwhelming"
 
+		fooddrink/alcoholic/romulale
+			name = "Romulale"
+			id = "romulale"
+			fluid_r = 20
+			fluid_g = 168
+			fluid_b = 254
+			alch_strength = 4
```

Comment:
way too high

Reply:
I just looked through the other drinks and picked out a value one of the other ones had that was on the higher end. I'm not sure exactly how they scale, around what would be a suitable strength to still be a "strong" drink while not being too high?

Reply:
alch strength of 1 is pretty much the upper limit of 'mundane' drinks;  ~0.8 would be pretty strong